contracts/BaseEscrowFactory.sol

```diff
@@ -46,11 +50,19 @@ abstract contract BaseEscrowFactory is IEscrowFactory, ResolverValidationExtensi
      * to a pre-computed deterministic address of the created escrow.
      * The external postInteraction function call will be made from the Limit Order Protocol
      * after all funds have been transferred. See {IPostInteraction-postInteraction}.
-     * `extraData` consists of:
```

Comment:
If we decide not to replace the detailed extraData layout comment with a link, let’s add an extra reference to its source, the `FeeTaker` contract in the `limit-order-settlement` repo.

This will make it easier to verify that the description stays in sync with the actual implementation and avoid silent desyncs in case the upstream logic changes.

Reply:
Got it. My point was only about the shared slice of `extraData`, not the whole structure.
- [`FeeTaker._getFeeAmounts()`](https://github.com/1inch/limit-order-protocol/blob/master/contracts/extensions/AmountGetterWithFee.sol#L64-L74)
- [Whitelist structure](https://github.com/1inch/fusion-protocol/blob/master/contracts/SimpleSettlement.sol#L113-L126)